corehq/apps/users/bulk_download.py

```diff
@@ -203,7 +203,10 @@ def make_web_user_dict(user, location_cache, domain):
 def make_invited_web_user_dict(invite, location_cache):
     location_codes = []
     try:
-        location_codes.append(location_cache.get(getattr(invite.location, 'location_id', None)))
+        location_codes.append(location_cache.get(getattr(invite.primary_location, 'location_id', None)))
```

Comment:
The only change here was the property rename `location` to `primary_location` so the existing behavior adds `None` to the list so I'm not expecting there to be an issue. I also tested it and didn't see an issue.

But i did catch a different bug while investigating this [1e10e35](https://github.com/dimagi/commcare-hq/pull/34550/commits/1e10e35029388f7a17855387f33776427ae433f6) so that was lucky

Reply:
I think it existed longer, i was looking at this https://github.com/dimagi/commcare-hq/blame/597a4f3f460c14368db47ff2649dfee0d56d63ef/corehq/apps/users/bulk_download.py#L203-L206 

Steph added this line`location_codes.append(location_cache.get(invite.supply_point))` 3 years ago, and `invite.supply_point` could be `None`